tests/tests_lite/strategies/test_registry.py

```diff
@@ -43,9 +43,7 @@ def __init__(self, param1, param2):
 
 def test_available_strategies_in_registry():
     expected = {
-        "ddp_sharded_find_unused_parameters_false",
         "ddp_sharded",
-        "ddp_find_unused_parameters_false",
```

Comment:
If we want, we can introduce the opposite flag, for setting `find_unused_parameters=True`.